addon/types/model_export/model.py

```diff
@@ -143,8 +144,8 @@ def generate_qc(self):
         if not self.reference and not self.stacking:
             return self.report(f'Unable to generate QC for: {self.name} (reference and stacking both not set)')
 
-        if not self.armature and not self.static:
-            return self.report(f'Unable to generate QC for: {self.name} (armature not set and static not enabled)')
+        # if not self.armature and not self.static:
+        #    return self.report(f'Unable to generate QC for: {self.name} (armature not set and static not enabled)')
```

Comment:
If your prop isn't static, you will probably want to animate it, by adding sequences, which use actions.
You can't have an action without something that is affected by it, such as an armature.
If you don't want to set an armature, make it a static prop.
Perhaps there is a use case for a prop that isn't static, doesn't need animations, and can live with only a default bone, but I can't imagine this would be used often.